packages/build/src/plugins_core/deploy/build_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+option go_package = "github.com/netlify/buildbot/buildservice/proto";
+
+package buildservice;
+
+// Service to communicate between Netlify Build and the buildbot
+service BuildService {
+  // Request the buildbot to deploy the site
+  rpc RunPostDeploy(PostDeployRequest) returns (PostDeployResponse) {}
+}
+
+message PostDeployRequest {
+  // Whether the request should wait for the deploy (including post-processing)
+  // to finish before completing
+  bool await_live = 1;
+}
+
+message PostDeployResponse {
+  // If false, `error` and `error_type` should be checked
+  bool succeeded = 1;
+  // Error message
+  string error = 2;
```

Comment:
Can we be more explicit and use `error_message`? (similar to `error_type`)

Reply:
Fixed :+1: